README.md

```diff
@@ -20,27 +20,39 @@
 [![Go Report Card][goreport-image]][goreport-url]
 [![Reference documentation][reference-documentation-image]][reference-documentation-url]
 
-This is the code repository for authd, an authentication daemon for cloud-based identity provider.
+Authd is an authentication daemon for cloud-based identity providers. It helps
+ensure the secure management of identity and access for Ubuntu machines
+anywhere in the world, on desktop and the server. Authd's modular design makes
+it a versatile authentication service that can integrate with multiple identity
+providers. MS Entra ID is currently supported and support for several other
```

Comment:
"supported/support" this is a bit redundant. Maybe change with

MS Entra ID is currently supported, and several other identity providers are under active development.

Reply:
Agreed.